internal/lsp/eval.go

```diff
@@ -138,21 +145,57 @@ func (l *LanguageServer) EvalWorkspacePath(
 	return EvalPathResult{Value: res, PrintOutput: hook.Output}, nil
 }
 
-func prepareRegoArgs(query ast.Body, bundles map[string]bundle.Bundle, printHook print.Hook) []func(*rego.Rego) {
+func prepareRegoArgs(
+	query ast.Body,
+	bundles map[string]bundle.Bundle,
+	printHook print.Hook,
+	cfg *config.Config,
+) []func(*rego.Rego) {
 	bundleArgs := make([]func(*rego.Rego), 0, len(bundles))
 	for key, b := range bundles {
 		bundleArgs = append(bundleArgs, rego.ParsedBundle(key, &b))
 	}
 
-	baseArgs := []func(*rego.Rego){
+	args := []func(*rego.Rego){
 		rego.ParsedQuery(query),
 		rego.Function2(builtins.RegalParseModuleMeta, builtins.RegalParseModule),
 		rego.Function1(builtins.RegalLastMeta, builtins.RegalLast),
 		rego.EnablePrintStatements(true),
 		rego.PrintHook(printHook),
 	}
 
-	return append(baseArgs, bundleArgs...)
+	args = append(args, bundleArgs...)
+
+	var caps *config.Capabilities
+	if cfg != nil && cfg.Capabilities != nil {
+		caps = cfg.Capabilities
+	} else {
+		caps = config.CapabilitiesForThisVersion()
+	}
+
+	var evalConfig config.Config
+	if cfg != nil {
+		evalConfig = *cfg
+	}
+
+	internalBundle := &bundle.Bundle{
+		Manifest: bundle.Manifest{
+			Roots:    &[]string{"internal"},
+			Metadata: map[string]any{"name": "internal"},
+		},
+		Data: map[string]any{
+			"internal": map[string]any{
+				"combined_config": config.ToMap(evalConfig),
+				"capabilities":    caps,
+			},
+		},
+	}
+
+	fmt.Fprintln(os.Stderr, "Loaded internal bundle")
```

Comment:
I guess this was left from testing :) I can have it removed in the next PR though.

Reply:
oops, indeed! thanks!